mobile/src/main/java/org/openhab/habdroid/ui/CloudNotificationListFragment.kt

```diff
@@ -61,6 +61,9 @@ class CloudNotificationListFragment : Fragment(), View.OnClickListener, SwipeRef
         val view = inflater.inflate(R.layout.fragment_notificationlist, container, false)
 
         recyclerView = view.findViewById(android.R.id.list)
+        emptyView = view.findViewById(android.R.id.empty)
```

Comment:
Couldn't you use https://kotlinlang.org/docs/tutorials/android-plugin.html#view-binding here?

Reply:
I started doing this, but the imports due to this technique got totally out of hand in WidgetAdapter due to the large number of different layouts involved. I therefore didn't use it there and for consistency reasons didn't use it elsewhere either.

Additionally, there are at least some people at Google who think it's a bad idea to use it. Interesting read in that regard: https://proandroiddev.com/the-argument-over-kotlin-synthetics-735305dd4ed0